### PR TITLE
fix(button): change to toggle in drawer in action header

### PR DESCRIPTION
**Pull request by matex1024**

### CODE REVIEW

File: src/frontend/components/app/action-header/styled-back-button.tsx [hunk 1]
```diff
@@ -1,41 +1,33 @@
...
-  const backButtonUrl = pathname
-    ? [pathname, search].join('')
-    : h.resourceUrl({ resourceId, search: window.location.search })
+  const navigate = useNavigate()
 
   return (
     <StyledLink
       size="icon"
-      to={backButtonUrl}
+      to={navigate(-1)}
```
Comment (dziraf) [inline src/frontend/components/app/action-header/styled-back-button.tsx:30]: How does it work currently? Didn't we decide to remove it completely from non-drawer ui?

**Review by dziraf — CHANGES_REQUESTED** [review]

File: src/frontend/components/app/action-header/styled-back-button.tsx [hunk 2]
```diff
@@ -1,41 +1,33 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Link as RouterLink } from 'react-router-dom'
+import { useNavigation, Link as RouterLink, useNavigate } from 'react-router-dom'
```
[the remark this answers is not in the document]
Comment (matex1024) [reply src/frontend/components/app/action-header/styled-back-button.tsx]: fixed

File: src/frontend/components/app/action-header/styled-back-button.tsx (hunk 1 above)
Comment (matex1024) [reply src/frontend/components/app/action-header/styled-back-button.tsx:28]: fixed

File: src/frontend/components/app/action-header/styled-back-button.tsx [hunk 3]
```diff
@@ -1,41 +1,33 @@
...
 } from '@adminjs/design-system'
 
-import { useSelector } from 'react-redux'
 import ViewHelpers from '../../../../backend/utils/view-helpers/view-helpers'
-import { ReduxState, RouterProps } from '../../../store'
 import allowOverride from '../../../hoc/allow-override'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const StyledLink = styled(({ rounded, ...rest }) => <RouterLink {...rest} />)<ButtonProps>`${ButtonCSS}`
 const h = new ViewHelpers()
```
[the remark this answers is not in the document]
Comment (matex1024) [reply src/frontend/components/app/action-header/styled-back-button.tsx]: fixed

**Review by dziraf — APPROVED** [review]

**Pull request merged**